lms/static/js/dashboard/dropdown.js

```diff
@@ -9,9 +9,15 @@ var edx = edx || {};
     // Generate the properties object to be passed along with business intelligence events.
     edx.dashboard.dropdown.toggleCourseActionsDropdownMenu = function(event) {
         // define variables for code legibility
-        var dashboardIndex = $(event.currentTarget).data().dashboardIndex,
-            $dropdown = $('#actions-dropdown-' + dashboardIndex),
-            dropdownButton = $('#actions-dropdown-link-' + dashboardIndex),
+        var $currentTarget = $(event.currentTarget),
+            dashboardIndex = $currentTarget.data().dashboardIndex,
+
```

Comment:
You can remove the white lines, var definitions should stack without spacing when comma separated.

Reply:
I just spaced them out like this for better readability. I will remove the lines.